x/feeabstraction/keeper/oracle.go

```diff
@@ -37,7 +37,10 @@ func (k Keeper) CalculateFeeTokenPrices(ctx sdk.Context) error {
 	// Find the price for the base token
 	baseTokenPrice, ok := twapPriceMap[params.NativeOracleDenom]
 	if !ok {
-		baseTokenPrice = params.FallbackNativePrice
+		// Disable fee abstraction if there is no pricing
+		k.Logger(ctx).Debug("akii has no price, feeabstraction disabled")
```

Comment:
Use the param when logging:

```suggestion
		k.Logger(ctx).Debug(fmt.Printf("%s has no price, feeabstraction disabled", params.NativeOracleDenom))
```

Reply:
Addressed